tests/render.rs

```diff
@@ -339,7 +339,7 @@ fn ticker_drop() {
                 .with_finish(ProgressFinish::AndLeave)
                 .with_message(format!("doing stuff {}", i)),
         );
-        new_spinner.enable_steady_tick(Duration::from_millis(50));
+        new_spinner.enable_steady_tick(Duration::from_millis(100));
```

Comment:
There was a test failure on Windows only, so I'm guessing that's just a timing artefact?

Reply:
Yes, I think so